### ISSUE TITLE
UX/accessibility Issue - Cannot reach the newsletter switches with tab on E-mail preferences

### ISSUE THREAD

**Issue by PereViki** [topcoder-platform/community-app#1404]
**Describe the bug**
All affordances should be available through tabbing through the site, however, newslettes switches cannot be reached from the keyboard.

**To Reproduce**
1. Go to 'https://www.topcoder-dev.com/settings/preferences'
2. Click on 'E-mail'
3. Tab through the page.

**Expected behavior**
The user should be able to set the newsletters from the keyboard.

**Screenshots**
http://www.screencast.com/t/ErcL4lSnY

**Desktop: MacBook Pro**
 - OS: Mac OS High Sierra
 - Browser: Chrome
 - Version: 69.0.3497.92

@ThomasKranitsas
